internal/pkg/dl/enrollment_api_key_integration_test.go

```diff
@@ -114,3 +114,32 @@ func TestSearchEnrollmentAPIKeyByPolicyID(t *testing.T) {
 		t.Fatal(diff)
 	}
 }
+
+func TestSearchEnrollmentAPIKeyByPolicyIDWithInactiveIDs(t *testing.T) {
```

Comment:
The previous test is deemed flakey (#1289), however it may be because that validates multiple returns; this test only does one so it should be OK.